task.sql

```diff
@@ -1,3 +1,13 @@
 USE ShopDB; 
 
 -- Create your stored procedure here
+DELIMITER //
+CREATE PROCEDURE get_warehouse_product_inventory(IN ID INT)
+BEGIN
+    SELECT Products.Name AS ProductName, ProductInventory.WarehouseAmount AS Amount
+    FROM ProductInventory
+    JOIN Products
+    ON ProductInventory.ProductID = Products.ID
+    WHERE ProductInventory.WarehouseID = ID;
+END //
+DELIMITER;
```

Comment:
![image](https://github.com/user-attachments/assets/481cc6ac-852b-493d-b15f-16339eb26321)

Problem with syntax in 13 line 


![image](https://github.com/user-attachments/assets/926f648d-719a-4889-97df-928f24f824cd)
Also, need to add Eof(End-of-file), need to add blank line at the end